meta-mender-commercial/recipes-monitor/monitor-client/monitor-client.inc

```diff
@@ -0,0 +1,28 @@
+LICENSE = "CLOSED"
+LICENSE_FLAGS = "commercial"
+
+RDEPENDS_${PN} = "bash"
+
+FILES_${PN} = " \
+    ${bindir}/monitord \
```

Comment:
This is a quite generic name. To avoid the risk of a future renaming, would it make sense to use "mender-monitord" instead?

Also, it is a bit confusing that this is called monitord, but the recipe is called monitor-client. But maybe this is best discussed face to face, remind me to bring it up in the standup.

Reply:
The name is somewhat provisional. We have [a task](https://tracker.mender.io/browse/MEN-4737) for it.

I agree with your you comments, though; the recipe name (and repository name) could be closer to the main binary or entry-point of the application.